packages/@ama-sdk/showcase-sdk/package.json

```diff
@@ -117,6 +117,6 @@
   "engines": {
     "node": "^18.19.1 || ^20.11.1 || >=22.0.0",
     "npm": ">=4",
-    "yarn": "<5.0.0"
+    "yarn": "<2.0.0"
```

Comment:
Why ?

Reply:
A miss from the postinstall process.
I will rollback it